contracts/Registry.vy

```diff
@@ -176,9 +176,9 @@ def _newProxyVault(
     guardian: address,
     name: String[64],
     symbol: String[32],
+    releaseTarget: uint256,
 ) -> address:
-    # NOTE: Underflow if no releases created yet (this is okay)
-    vault: address = create_forwarder_to(self.releases[self.nextRelease - 1])  # dev: no releases
+    vault: address = create_forwarder_to(self.releases[releaseTarget])
```

Comment:
what happens if releaseTarget doesnt exist?

Reply:
It should fail when it does checks further down the line, but easy to check upfront

Reply:
nvm, i noticed this is internal so the delta protects against using release targets that we don't have

Reply:
fixed